{{cookiecutter.project_slug}}/merge_production_dotenvs_in_dotenv.py

```diff
@@ -0,0 +1,65 @@
+import os
```

Comment:
What is this file for?

Reply:
To merge multiple `.envs/.production/*` files in a specific order in a single `.env`; effectively, this is supposed to be done manually ones any of the `.envs/.production/*` changes.

Reply:
Since we won't be providing `env.example`/`.env` out-of-box anymore I thought that would the easiest way to let the client have an up-to-date version of `.env` throughout the project lifecycle.